packages/former/src/Playground.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="flex w-full h-screen bg-gray-100">
+  <div class="flex w-full h-screen dark:bg-zinc-900 bg-gray-100 dark:text-zinc-100">
```

Comment:
@linusbbb It is totally sufficient to define the text-color here on top because the color is inherited to all sub components although they overwrite the color. But this nowhere the case so please remove all other `dark:text-zinc-100` occurences